src/neuroconv/datainterfaces/ophys/miniscope/miniscopeimagingdatainterface.py

```diff
@@ -74,11 +76,13 @@ def get_metadata(self) -> DeepDict:
         return metadata
 
     def get_metadata_schema(self) -> dict:
+        """Get the metadata schema for the Miniscope imaging data."""
         metadata_schema = super().get_metadata_schema()
         metadata_schema["properties"]["Ophys"]["definitions"]["Device"]["additionalProperties"] = True
         return metadata_schema
 
     def get_original_timestamps(self) -> np.ndarray:
+        """Get the original timestamps from the Miniscope data."""
```

Comment:
Needs a Returns section

Reply:
Same, noqa and should inherit from base class.